changelogs/fragments/58973-luks_device_add-type-option.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- luks_device - added the `type` option that allows user explicit define the LUKS container format version (can be `luks1` or `luks2`)
```

```suggestion
- luks_device - added the ``type`` option that allows user explicit define the LUKS container format version
```
1. Changelog fragments are [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText), i.e. use double backticks for code formatting.
2. Changelog shouldn't explain changes in too much details; users can consult the module documentation to find out how to actually do it :)

---

Hmm. thanks for the tips.